frontend/dockerfile/docs/reference.md

```diff
@@ -1053,11 +1053,11 @@ image, consider setting a value for a single command instead:
 RUN DEBIAN_FRONTEND=noninteractive apt-get update && apt-get install -y ...
 ```
  
-Or using [`ARG`](#arg), which is not persisted in the final image:
+Or parametrized using [`ARG`](#arg), which is not persisted in the final image:
 
 ```dockerfile
 ARG DEBIAN_FRONTEND=noninteractive
-RUN apt-get update && apt-get install -y ...
+RUN DEBIAN_FRONTEND=$DEBIAN_FRONTEND apt-get update && apt-get install -y ...
```

Comment:
This is not needed, `ARG` are always available as environment variable at build time in `RUN` instructions.

Reply:
I was surprised to see it so I tested it. It did not work. I tested it again and it works just fine. Even with legacy Docker build (which I suspected I might have used)..

No idea why I failed to use it at first. We can close this. :-)